packages/eslint-plugin-js/rules/padding-line-between-statements/padding-line-between-statements.ts

```diff
@@ -337,7 +337,7 @@ const StatementTypes = {
   'block': newNodeTypeTester('BlockStatement'),
   'empty': newNodeTypeTester('EmptyStatement'),
   'function': newNodeTypeTester('FunctionDeclaration'),
-
+  'ts-method': newNodeTypeTester('TSMethodSignature'),
```

Comment:
ts specific logic should be handled in the ts plugin, there you are touching the js one

Reply:
Ok. I will fix it